crates/trie/common/src/proofs.rs

```diff
@@ -119,7 +120,7 @@ impl StorageMultiProof {
 }
 
 /// The merkle proof with the relevant account info.
-#[derive(PartialEq, Eq, Debug)]
+#[derive(Clone, PartialEq, Eq, Debug, Serialize, Deserialize)]
 pub struct AccountProof {
```

Comment:
can we do rename_all camel case?

Reply:
Oops, sorry should have checked other places where this is derived. Fixed now.